### PR TITLE
[FEATURE] ui5 serve: Add serve-csp-reports option

### PR DESCRIPTION
**Pull request by tobiasso85**
Add option to use csp-report-path to create csp reports for server.

See: SAP/ui5-server/pull/323

Sample call
```bash
ui5 serve --serve-csp-reports
```

### CODE REVIEW

**Comment by matz3** [comment]
Looks good, but maybe change the commit message to `[FEATURE] ui5 serve: Add csp-report-path option`.

**Comment by tobiasso85** [comment]
> Looks good, but maybe change the commit message to `[FEATURE] ui5 serve: Add csp-report-path option`.

changed the commit messages accordingly

File: lib/cli/commands/serve.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ serve.builder = function(cli) {
 			default: false,
 			type: "boolean"
 		})
+		.option("serve-csp-reports", {
+			describe: "collects and serves csp-reports upon request to '/.ui5/csp/csp-reports.json'",
```
Comment (RandomByte) [inline lib/cli/commands/serve.js:54]:
```suggestion
			describe: "Collects and serves CSP reports upon request to '/.ui5/csp/csp-reports.json'",
```

File: test/lib/cli/commands/serve.js [hunk 2]
```diff
@@ -343,3 +347,22 @@ test.serial("ui5 serve: --framework-version", async (t) => {
 		}
 	}]);
 });
+
+test.serial("ui5 serve: --serve-csp-reports=true", async (t) => {
```
Comment (RandomByte) [inline test/lib/cli/commands/serve.js:351]:
```suggestion
test.serial("ui5 serve --serve-csp-reports", async (t) => {
```

Like in the other tests: No colon and boolean arguments don't need a value

File: lib/cli/commands/serve.js (hunk 1 above)
Comment (tobiasso85) [reply lib/cli/commands/serve.js:54]: done

File: test/lib/cli/commands/serve.js (hunk 2 above)
Comment (tobiasso85) [reply test/lib/cli/commands/serve.js:351]: done

**Pull request merged**